From 2c8ef413434cfc3dcdc8c617a3f76f7114351b95 Mon Sep 17 00:00:00 2001
From: Nizan Zorea <nzorea@marvell.com>
Date: Thu, 24 Mar 2016 14:09:35 +0200
Subject: [PATCH 0976/1240] fix: fdt_config: add parentheses around MACRO
 EEPROM_STRUCT_SIZE

- without the parentheses A-3700 failed since it read the
  wrong size.
- this patch fixed the macro so the calculation will be correct.

Change-Id: I359f6db53e80b5ea326386b78895db671021afdb
Signed-off-by: Nizan Zorea <nzorea@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28503
Reviewed-by: Haim Boot <hayim@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 board/mvebu/common/cfg_eeprom.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/mvebu/common/cfg_eeprom.h b/board/mvebu/common/cfg_eeprom.h
index 75310f9..c534ef5 100644
--- a/board/mvebu/common/cfg_eeprom.h
+++ b/board/mvebu/common/cfg_eeprom.h
@@ -69,7 +69,7 @@ struct config_types_info {
 
 #define BOARD_ID_INDEX_MASK		0x10
 #define I2C_PAGE_WRITE_SIZE		32
-#define EEPROM_STRUCT_SIZE		sizeof(struct eeprom_struct) - sizeof(board_config_val.fdt_blob)
+#define EEPROM_STRUCT_SIZE		(sizeof(struct eeprom_struct) - sizeof(board_config_val.fdt_blob))
 #define HW_INFO_MAX_PARAM_NUM		32
 struct hw_info_point_struct {
 	char *name;
-- 
1.9.1

